lib/strategies/base.js

```diff
@@ -53,6 +53,10 @@ BaseStrategy.prototype._attachListeners = function () {
     self._reformatInput(event);
   });
   self.inputElement.addEventListener('input', function (event) {
+    // Safari AutoFill fires CustomEvents -- mark the input as unformatted without actually unformatting the current value
```

Comment:
Literally `CustomEvent` type events. I could update this to `fires CustomEvent events`?